Website/htdocs/mpmanager/app/Listeners/ClusterMetaDataListener.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Listeners;
+
+use App\Events\ClusterMetaDataEvent;
+use App\Models\Address\Address;
+use App\Models\ClusterMetaData;
+use App\Models\Meter\MeterParameter;
+use App\Models\Person\Person;
+use Illuminate\Events\Dispatcher;
+
+class ClusterMetaDataListener
+{
+    public function incrementConnectedMetersCount(MeterParameter $meterParameter): void
```

Comment:
@buralp  `increase` would be a better naming. Please change all `increment` 's with  `increase` .